Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7
+RUN pip install poetry
+WORKDIR /turbot
+COPY . ./
+RUN poetry config virtualenvs.create false \
+    && poetry install --no-ansi --no-interaction
```

Comment:
You probably also want to add `--no-dev` to reduce docker image size. The dev dependencies are only required for running the scripts and test suite.